### PR TITLE
py>: operator supports python option arguments

### PR DESCRIPTION
**Pull request by sonots**
Python version of https://github.com/treasure-data/digdag/pull/1022

FYI: It seems that `sh>` operator already supports List.

~I did not add tests because I could not find any existing tests for python operator, but I verified this works manually.~

### CODE REVIEW

**Comment by muga** [comment]
Thank you for suggesting. I will take a look https://github.com/treasure-data/digdag/pull/1022 first.

**Comment by muga** [comment]
https://github.com/treasure-data/digdag/pull/1027 is for `rb>` operator but, it . would be helpful for your `py>` operator testing.

**Comment by muga** [comment]
https://github.com/treasure-data/digdag/pull/1022 looks good. Can I ask you to change this according to it?

**Comment by sonots-zozo** [comment]
Added tests.

**Comment by muga** [comment]
Looks good. Thank you for adding tests. 

@yoyama what do you think?

**Comment by yoyama** [comment]
I commented on a very minor issue. After fix it I would like to merge it.

**Comment by sonots** [comment]
Where is the comment? (Maybe, you need to publish your review comments)

File: digdag-tests/src/test/java/acceptance/PyIT.java [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+                .port(server.port())
+                .build();
+    }
+
+    @Test
+    public void verifyDefaultConfigurationParamsAreNotUploadedIfConfigurationFileIsSpecified()
+            throws Exception
+    {
+        Path tempdir = folder.getRoot().toPath().toAbsolutePath();
+        Path configDir = folder.getRoot().toPath().toAbsolutePath();
```
[the remark this answers is not in the document]
Comment (yoyama) [reply digdag-tests/src/test/java/acceptance/PyIT.java]: @sonots as you mentioned, it was not published.

[the remark this answers is not in the document]
Comment (sonots) [reply digdag-tests/src/test/java/acceptance/PyIT.java]: Thanks, removed.

[the remark this answers is not in the document]
Comment (yoyama) [reply digdag-tests/src/test/java/acceptance/PyIT.java]: Sorry I can't confirm update.

[the remark this answers is not in the document]
Comment (sonots) [reply digdag-tests/src/test/java/acceptance/PyIT.java]: Oops, pushed.

**Pull request merged**